lib/Limitation/Mapper/GroupLimitationMapper.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Limitation\Mapper;
+
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class GroupLimitationMapper extends MultipleSelectionBasedMapper
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    protected function getSelectionChoices()
+    {
+        return [
+            0 => $this->translator->trans('policy.limitation.group.any', [], 'ezrepoforms_role'),
+            1 => $this->translator->trans('policy.limitation.group.self', [], 'ezrepoforms_role'),
```

Comment:
@lolautruche Not strictly needed to enforce the keys, but IMHO being verbose here has value for others reading the code - the keys correspond to specific values in the limitationtype code. Might consider if we should use constants, though.